modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewRelatedEntriesDisplayContext.java

```diff
@@ -222,7 +225,13 @@ public List<SelectOption> getSelectOptions() {
 				StringPool.BLANK));
 
 		for (CTCollection ctCollection : ctCollections) {
-			if (ctCollection.getCtCollectionId() != _ctCollectionId) {
+			boolean canUpdate = CTCollectionPermission.contains(
+				PermissionThreadLocal.getPermissionChecker(), ctCollection,
+				ActionKeys.UPDATE);
+
+			if ((ctCollection.getCtCollectionId() != _ctCollectionId) &&
+				canUpdate) {
```

Comment:
@ksuetani 
Can you remove `canUpdate` and just add the logic inline in the if statement. BChan doesn't like declaring variables that are used only once.